sap/5.9/modules/ROOT/pages/sap-connector-config-snc.adoc

```diff
@@ -51,6 +52,54 @@ You can use STRUST to see names that are already in use.
 +
 `sapgenpse <export_own_cert> –v –p RFC –o RFC_MULE.crt`
 
+[[configure-the-client-environment-when-deploying-in-cloudhub-2]]
+== Configure the Client Environment When Deploying in CloudHub 2.0
+
+In SAP Connector 5.9.5 and later, an application that uses the Certificate connection type can be deployed in CloudHub 2.0. SAP Connector has an embedded installer that sets up the Personal Security Environment (PSE).
+
+You must first create your application:
+
+. Open Anypoint Studio.
+. Navigate to the `src/main/resources` folder of your application.
+. In the `src/main/resources` folder, create a folder named `sec`.
++
+image::sap-resources-sec.png[Sap Resources Sec Folder]
++
+. Add the following files in the `sec` folder:
+* `libsapcrypto.so`
++
+The Linux version of the SAP CommonCryptoLib.
+* `sapgenpse`
++
+The Linux version of the SAP command line tool that manages PSE files.
+* `RFC.pse`
++
+The file you created in the <<configure-the-client-environment, previous step>>.
+. Because CloudHub 2.0 runs on Linux machines, ensure that any dependency of the application is compiled for Linux, including `com.sap.conn.jco:libsapjco3`.
+. Save the application and export the `jar` file that is used for CloudHub 2.0 deployment.
+
+You must then import the created application in CloudHub 2.0:
+
+. Log in to your Anypoint Platform account and navigate to Runtime Manager.
+. Click on *Deploy Application*.
+. In *Application Name*, enter a name for the application, for example, `cert-ch2`.
+. In *Deployment Target*, select `CloudHub 2.0` if it is not selected already.
+. Click *Choose File* and upload your `jar` file.
+. In the *Properties* tab, add the following:
++
+[%header,cols="30a,30,30"]
+|===
+|Property|Example Value|Description
+|`secFilesCopy`|`true`|Triggers the embedded installer.
+|`sourceSecFolder`|`/opt/mule/apps/cert-ch2/sec/`|This is where the `resources/sec` folder is located in CloudHub 2.0. Replace the name of the application to match your scenario.
+|`targetSecFolder`|`/tmp/sec/`|This value should remain hardcoded.
+|`pseFile`|`RFC`|The name of the PSE file, without an extension.
+|`psePassword`|`Your PSE file password`|The PSE file password. You can click on the *Protect* button next to this property to obfuscate its content.
```

Comment:
```suggestion
|`psePassword`|`Your PSE file password`| PSE file password. You can click on the *Protect* button next to this field to hide its content.
```

Reply:
According to the Style Guide, we are supposed to use "Click xxx" instead of "Click on xxx." (I am probably guilty of using "Click on" myself.) I won't mark these going forward.